associates/jennifer-lisa.json

```diff
@@ -0,0 +1,5 @@
+{
+	"first": "Jennifer",
+	"last": "Lisa"
```

Comment:
Oops, there's  missing comma here. Can you please add that, commit it, then push to your branch again?